src/test/scala/mesosphere/marathon/benchmarks/storage/zk/ZkStorageBenchmark.scala

```diff
@@ -1,173 +0,0 @@
-package mesosphere.marathon.benchmarks.storage.zk
```

Comment:
Scala meter sucks. I have something else coming down the pipeline for benchmarks- plus it integrates with velocity